core/core_bind.cpp

```diff
@@ -557,6 +558,17 @@ String OS::get_unique_id() const {
 	return ::OS::get_singleton()->get_unique_id();
 }
 
+void OS::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	String pf = p_function;
+	if (p_idx == 0 && pf == "has_feature") {
+		HashSet<String> presets = EditorExport::get_singleton()->get_available_features();
```

Comment:
```suggestion
		HashSet<String> presets;
#ifdef TOOLS_ENABLED
		presets = EditorExport::get_singleton()->get_available_features();
#endif
```
Or the entire block, or method entirely